.storybook/webpack.config.js

```diff
@@ -32,6 +32,17 @@ module.exports = {
         test: /\.css$/,
         use: ['style-loader', 'css-loader'],
       },
+      {
```

Comment:
these changes are just a duplication of what we do in the primary webpack config file, since we need the same support in storybook land.

Reply:
👍